frontend/package.json

```diff
@@ -41,5 +42,8 @@
       "last 1 safari version"
     ]
   },
-  "author": "CodersCamp 2020 | Tomasz Dworniczak, Anna Lamperska, Piotr Rynio, Paweł Szambelan, Dominika Zwolenik, Mateusz Nowak"
+  "author": "CodersCamp 2020 | Tomasz Dworniczak, Anna Lamperska, Piotr Rynio, Paweł Szambelan, Dominika Zwolenik, Mateusz Nowak",
+  "devDependencies": {
+    "@types/jest": "^26.0.21"
```

Comment:
Czemu akurat to zostało przeniesione? Bo w sumie wszystkie @types nie sa potrzebne do działania i moglyby byc w devDependencies.

Reply:
Po usunięciu pliku frontend/src/react-app-env.d.ts przestały działać testy, próbowałam to naprawić. Szukałam różnych rozwiązań w internecie, to było jedno z nich. Finalnie wszystko po restarcie działało